chicago/permit_cleaning.py

```diff
@@ -470,9 +471,9 @@ def add_address_link_and_suggested_pins(df, chicago_pin_universe):
         df["Applicant Street Address* [ADDR1]"],
     )
 
-    # Suggested PINs (replace NA with NO PIN FOUND)
+    # Suggested PINs (replace NA with empty string)
     df = df.rename(columns={"pin": "Suggested PINs"})
-    df["Suggested PINs"] = df["Suggested PINs"].fillna("NO PIN FOUND")
+    df["Suggested PINs"] = df["Suggested PINs"].fillna("")
```

Comment:
I think the empty string makes it easier to distinguish between cells with and without PIN matches when scanning multiple rows, but ultimately it's a personal preference thing, so we'll make sure to ask Data Integrity what they think about this choice when we present the file to them.